### ISSUE TITLE
WPF TextBox XamlParseException when culture is not "en-US" (.NET Core 3.0.0 Preview 6)

### ISSUE THREAD

**Issue by vitidev** [dotnet/wpf#965]
# WPF TextBox XamlParseException when culture is "ru-RU" (.NET Core 3.0.0 Preview 6)

Empty netcore wpf project and window with textbox

```c#
    public MainWindow()
    {
         InitializeComponent(); // System.Windows.Markup.XamlParseException
    }
```

```xml
    <Window
        x:Class="WpfApp1.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:local="clr-namespace:WpfApp1"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        Title="MainWindow"
        Width="800"
        Height="450"
        mc:Ignorable="d">
        <TextBox />
    </Window>
```

Exception details (localized (ru))
```
    System.Windows.Markup.XamlParseException
      HResult=0x80131501
      Message="Вызов конструктора для типа "System.Windows.Controls.TextBox", удовлетворяющего указанным ограничениям привязки, привел к выдаче исключения.": номер строки "13" и позиция в строке "10".
      Source=PresentationFramework
      StackTrace:
       at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
       at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
       at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
       at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
       at System.Windows.Application.LoadComponent(Object component, Uri resourceLocator)
       at WpfApp1.MainWindow.InitializeComponent() in F:\dev\test\WpfApp1\WpfApp1\MainWindow.xaml:line 1

    Inner Exception 1:
    ArgumentException: Requested value 'УДАЛИТЬ' was not found.
```

but it works with en-US culture

```c#
    public MainWindow()
    {
        Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-us");
        InitializeComponent();
    }
```

ps: .NET Core 3.0.0 Preview 5 - everything works well

**Comment by rladuca** [comment]
FYI @ryalanms

**Comment by jose8789** [comment]
I have found the same error with portuguese culture

**Comment by AlexeiScherbakov** [comment]
I also can confirm 'Requested value 'УДАЛИТЬ' was not found' exception on Russian locale

**Comment by grubioe** [comment]
This was fixed via #919 
If using Preview 6: Workaround for this by adding this property to project file:
```xml
<PropertyGroup>
  <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
</PropertyGroup>
```

**Comment by vaibhavk69** [comment]
following is the code followed by issue that  i'm facing right now can someone help me:
<Window
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfApp1"
        xmlns:DataAnnotations="clr-namespace:System.ComponentModel.DataAnnotations;assembly=System.ComponentModel.Annotations" x:Class="WpfApp1.MainWindow"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Grid Margin="-323,-194,-209,-213">
        <Grid.Background>
            <ImageBrush ImageSource="/best-government-hospitals-bangalore-1200x870.png"/>
        </Grid.Background>
        <TextBox TextWrapping="Wrap" Margin="233,92,994,720" FontSize="20" TextChanged="TextBox_TextChanged_1" Opacity="0.25" Text="User Name">
            <TextBox.DataContext>
                <DataAnnotations:EmailAddressAttribute/>
            </TextBox.DataContext>
        </TextBox>
        <TextBox HorizontalAlignment="Left" Margin="969,536,0,0" Text="TextBox" TextWrapping="Wrap" VerticalAlignment="Top" Width="120"/>
        <TextBox Text="Password" TextWrapping="NoWrap" Margin="233,172,994,644" TextChanged="TextBox_TextChanged" FontSize="20" Opacity="0.25" MaxLength="14" MaxLines="1" Cursor="Pen">
            <TextBox.DataContext>
                <PasswordBox BorderThickness="1,1,1,1" CacheMode=""/>
            </TextBox.DataContext>
            <TextBox.Template>
                <ControlTemplate/>
            </TextBox.Template>
        </TextBox>
        <Label Content="ID" HorizontalAlignment="Left" Margin="50,92,0,0" VerticalAlignment="Top" Height="69" Width="117" FontSize="20"/>
        <Label Content="Password" HorizontalAlignment="Left" Margin="50,166,0,0" VerticalAlignment="Top" Height="31" Width="117" FontSize="20"/>
        <Button Content="LOG IN" HorizontalAlignment="Left" Margin="233,241,0,0" VerticalAlignment="Top" Height="32" Width="80"/>

    </Grid>
</Window>


error message:

System.Windows.Markup.XamlParseException
  HResult=0x80131501
  Message='Failed to create a 'CacheMode' from the text ''.' Line number '22' and line position '30'.
  Source=PresentationFramework
  StackTrace:
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadComponent(Object component, Uri resourceLocator)
   at WpfApp1.MainWindow.InitializeComponent() in C:\Users\vaibh\source\repos\WpfApp1\WpfApp1\MainWindow.xaml:line 1
   at WpfApp1.MainWindow..ctor() in C:\Users\vaibh\source\repos\WpfApp1\WpfApp1\MainWindow.xaml.cs:line 25

  This exception was originally thrown at this call stack:
    [External Code]

Inner Exception 1:
FormatException: Token is not valid.
